cheatsheets/Ruby_on_Rails_Cheatsheet.md

```diff
@@ -475,11 +455,11 @@ In general, it is important to have a process for updating dependencies. An exam
 
 Use [brakeman](https://brakemanscanner.org/), an open source code analysis tool for Rails applications, to identify many potential issues. It will not necessarily produce comprehensive security findings, but it can find easily exposed issues. A great way to see potential issues in Rails is to review the brakeman documentation of warning types.
 
-There are emerging tools that can be used to track security issues in dependency sets, like [this](https://appcanary.com) and [this](https://gemnasium.com).
+There are emerging tools that can be used to track security issues in dependency sets, like [this](https://github.blog/2017-11-16-introducing-security-alerts-on-github/) and [this](https://docs.gitlab.com/ee/user/application_security/dependency_scanning/).
```

Comment:
Instead of using the term `this`, can you instead say what it is? so `like [security alerts on github]` maybe?

Reply:
Good call, updated